Source/Core/Core/IOS/Network/KD/NetKDRequest.h

```diff
@@ -52,11 +51,24 @@ class NetKDRequestDevice : public Device
     return std::nullopt;
   }
 
+  enum class ErrorType
+  {
+    ACCOUNT,
```

Comment:
We don't use SCREAMING_CASE for enum class values (because unlike C-style enumerators they do not leak into the surrounding scope, so there is less of a need for them to stand out)